src/axom/sidre/docs/sphinx/file_io.rst

```diff
@@ -50,7 +50,23 @@ which omits error-checking and recovery for brevity and clarity.
 The ``loadExternalData()`` method is used to read "external" data from an HDF5 
 file created with the ``sidre_hdf5`` protocol.  This is data referred to by 
 *external* views. Such views refer to data that is not stored in Sidre buffers 
```

Comment:
Wasnt from the PR but there is two differing ways to highlight "external" here... italicized and quoted.